opm/utility/ECLGraph.cpp

```diff
@@ -274,6 +279,19 @@ namespace {
                 std::vector<T>
                 scatterToGlobal(const std::vector<T>& x) const;
 
+                /// Map input vector to active grid cells.
+                ///
+                /// \param[in] x Input vector, defined on the explicitly
+                ///              active cells, all global cells or some
+                ///              other subset (e.g., all non-neighbouring
+                ///              connections).
+                ///
+                /// \return Input vector mapped to active cells or unchanged
```

Comment:
> OK, so the restart data vectors can contain cell-wise vectors on arbitrary sets of cells?

Yes, almost arbitrary.  The most common case is pore-volume on global cells, remaining (floating-point) vectors on active, but we must be prepared for pretty much anything.